### PR TITLE
Show source count in search frontpage info boxes

### PR DESCRIPTION
**Pull request by hypesystem**

### CODE REVIEW

File: src/app/search/simple/search-simple-resolver.service.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<SourceCounts> {
+    return this.http.get<any>(`${environment.apiUrl}/stats`)
+      .pipe(map((result) => {
+        return {
+          personAppearanceCount: result.esPersonAppearanceCount,
+          lifecourseCount: result.esLifecourseCount,
+        };
+      }));
+  }
+}
```
Comment (Arth101) [inline src/app/search/simple/search-simple-resolver.service.ts:29]: Missing newline eof

File: src/app/search/simple/search-simple.component.ts [hunk 2]
```diff
@@ -112,4 +121,10 @@ export class SimpleSearchComponent implements OnInit {
       }
     });
   }
+
+  enhanceText(text: String) {
```
Comment (Arth101) [inline src/app/search/simple/search-simple.component.ts:125]: Maybe more like "formatText()" but I cant seem to find a good name myself :sweat_smile:

**Review by Arth101 — APPROVED** [review]
Looks good irl :). Minor changes requested

File: src/app/search/simple/search-simple.component.ts (hunk 2 above)
Comment (hypesystem) [reply src/app/search/simple/search-simple.component.ts:125]: I like enhance better for now - we're not formatting anything but adding content to it? Dunno what the better name is either, but I'm open for one if we find it.

**Pull request merged**